src/coreclr/jit/gentree.cpp

```diff
@@ -11702,6 +11702,66 @@ GenTree* Compiler::gtFoldExprCall(GenTreeCall* call)
     return call;
 }
 
+#ifdef FEATURE_HW_INTRINSICS
+
+//------------------------------------------------------------------------
+// gtFoldHWIntrinsic: see if a HWIntrinsic is foldable
+//
+// Arguments:
+//    node - HWIntrinsic node to examine
+//
+// Returns:
+//    The original node if no folding happened.
+//    An alternative tree if folding happens.
+//
+// Notes:
+//    Checks for HWIntrinsic nodes to Vector64.Create/Vector128.Create/Vector256.Create,
+//    and if the call is to one of these, attempt to optimize.
+
```

Comment:
```suggestion
//
```
Makes VS show the comment on hovering over a function call.